roborazzi-junit-rule/build.gradle

```diff
@@ -39,7 +39,9 @@ android {
 dependencies {
   implementation project(':roborazzi')
   implementation libs.androidx.test.ext.junit.ktx
+  compileOnly libs.robolectric
   compileOnly libs.androidx.test.ext.junit.ktx
   compileOnly libs.androidx.compose.ui.test
   compileOnly libs.androidx.compose.ui.test.junit4
+  api libs.accessibility.test.framework
```

Comment:
I was thinking we could use `compileOnly` for this library because it has a lot of dependencies. Alternatively, we could create a module for this, like `roborazzi-accessibility-checks`, but managing it might be a bit challenging.  
While `compileOnly` would be great, we might accidentally use the APIs in `roborazzi` module, leading to errors. So, I’m considering adding a module like `roborazzi-accessibility-check`.  
I’ve committed the changes to separate the module. Could you check it out?
https://github.com/yschimke/roborazzi/compare/add_a11y_checks...takahirom:roborazzi:takahirom/separate-roborazzi-accessibility-check/2024-11-17

Reply:
Merged into this PR.  Thanks.